tests/builder/require.rs

```diff
@@ -41,6 +41,18 @@ fn option_required() {
     assert_eq!(err.kind(), ErrorKind::MissingRequiredArgument);
 }
 
+#[test]
+fn option_required_and_conflicts_with_other_option() {
+    let result = Command::new("cli")
+        .arg(arg!(brick: -b "do something harmful"))
+        .arg(arg!(fix: -f "do something good").conflicts_with("brick"))
+        .arg(arg!(dry_run: -d "don't do it Louis").requires("fix"))
+        .try_get_matches_from(vec!["", "-b", "-d"]);
+    assert!(result.is_err());
+    let err = result.err().unwrap();
+    assert_eq!(err.kind(), ErrorKind::MissingRequiredArgument);
+}
```

Comment:
As I would prefer every commit to pass tests, please make this test pass with the bad behavior and then the followup commit would make it pass with the good behavior